packer/ccmarketplace.pkr.hcl

```diff
@@ -57,6 +57,12 @@ build {
     destination = "/tmp/install_cli.py"
   }
 
+provisioner "file" {
```

Comment:
let's remove the cyclecloud_local/ dir 

Instead use:
provisioner "file" {
  source      = "../${var. cyclecloud_package_name}" != "../" ? "../${var. cyclecloud_package_name}" : "../README.md"
  destination = "/tmp/${var.cyclecloud_package_name}"
}